netlify.toml

```diff
@@ -1,2 +1,2 @@
 [build]
-  ignore = "git diff --quiet $COMMIT_REF $CACHED_COMMIT_REF -- docs/ www/"
```

Comment:
The Netlify build doesn’t update if there’s been a core change to Astro 😕. That’s why the preview doesn’t reflect the core change anymore.

Reply:
Should be fine!